app/Helpers/Helper.php

```diff
@@ -65,6 +65,22 @@ function durationToSpan($duration): String {
     return $return;
 }
 
+function nextStation(&$status) {
+    $stops = json_decode($status->trainCheckin->HafasTrip->stopovers);
+    $nextStopIndex = count($stops) - 1;
+
+    // Wir rollen die Reise von hinten auf, damit der nächste Stop als letztes vorkommt.
+    for ($i=count($stops)-1; $i > 0; $i--) {
+        $arrival = $stops[$i]->arrival;
+        if($arrival != null && strtotime($arrival) > time()) {
+            $nextStopIndex = $i;
+            continue;
+        }
+        break; // Wenn wir diesen Teil der Loop erreichen, kann die Loop beendert werden.
+    }
+    return $stops[$nextStopIndex]->stop->name;
+}
+
```

Comment:
Da das eh mit einem Status-Objekt arbeitet, würde ich die Methode gerne entweder im StatusController (als statische Methode) oder noch lieber im Status-Model sehen.

Das Helper-File war sonst nur für Built-In-Typen (DateTimes, Strings) etc reserviert.